main.js

```diff
@@ -137,9 +138,15 @@ function prepareDataset () {
     })
 
     dataset.departements[name] = departement
-  })
+  }).then(() => dataset)
```

Comment:
It is a little bit strange to return here the variable that was captured by the closure. 
Can't we use `Promise.map` to return an array, that is returned through the Promise chain, and then the `dataset` variable is returned? 
Otherwise it's ok for me if it works 😄